dist/dist-packages/linux/linux.test.bash

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
```

Comment:
there's a fair bit of duplicated code in the docker test as here. can you extract this into two steps?
a.) install the things
b.) test the things

Then the "test the things" section would be identitcal and reusable across any / all deployment mechanism

Reply:
That sounds great but I only see one redundant line:

```bash
go test -v -count=1 -tags="quickstart manual" ./ziti/cmd/edge/...
```